modules/migration-prerequisites.adoc

```diff
@@ -24,6 +23,28 @@ endif::[]
 ** The clusters must be able to communicate using OpenShift routes on port 443.
 ** The clusters must have no critical conditions.
 ** The clusters must be in a ready state.
+** The appropriate firewall rules must be configured on the target cluster.
+** Load balancing must be correctly configured on the target cluster.
+
+* Resources:
+
+** Check whether your application uses a service network or an external route to communicate with services and make sure the relevant route exists.
+** If your application uses non-namespaced resources, you must re-create them on the target cluster.
```

Comment:
That's a little unclear. Are you referring to cluster-scoped resources? This should be checked with an SME.